gammapy/maps/hpxnd.py

```diff
@@ -648,3 +649,28 @@ def get_angle(x, t):
         ax.coords.grid(color='w', linestyle=':', linewidth=0.5)
 
         return fig, ax, p
+
+    def make_region_mask(self, region, inside=True):
```

Comment:
I think this code is 100% identical in WCS and HPX case.
Suggest to avoid the code duplication and move it to the base class.

Reply:
A priori no because you have no constructor on the base class.
I suppose we can do it using `Map.from_geom()`and then assign `Map.data=mask`